README.md

```diff
@@ -0,0 +1,74 @@
+# Compass Components
+
+The compass components package is the implementation of the [compass design system](https://zeroheight.com/29be2c109/p/995619-compass-design-system) meant (but not limited) to be used in the mattermost products: Messaging, Playbooks and Focalboard.
+
+The intention behind the püackage is to create a unified look and feel across the products.
+
+## Use compass components in your project
+
+1. install compass components from npm
+
+    in your project run
+
+    ```bash
+    npm i @mattermost/compass-components
+    ```
+
+2. import `ThemeProvider` in your application root
+
+    In order for the compass components to be syled correctly you need to wrap your applcation in a `ThemeProvider`.
+
+    It also manages custom themes for your application (if you wish so).
+
+    ```javascript
+    import ThemeProvider from '@mattermost/compass-components/utilities/theme';
```

Comment:
optional: we can show an example of how to wrap their application as well?

Reply:
Will add this on Monday 👍🏻